src/main/java/io/swagger/oas/inflector/OpenAPIInflector.java

```diff
@@ -358,7 +358,17 @@ protected void init(Configuration configuration) {
                 }
             }
             final Resource.Builder builder = Resource.builder();
-            builder.path(basePath(originalBasePath, config.getSwaggerBase() + "debug.json"))
```

Comment:
when `config.getSwaggerBase()` returns a value, it does not have a trailing `/` and we end up with a mashed `swaggerdebug.json` as the path in the integration tests.